src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -892,6 +921,58 @@ bool isObjectInitializerInitializedObjectTarget(IFlowCaptureReferenceOperation r
                 } && left == referenceSyntax;
             }
 
+            bool isInterpolatedStringArgumentCapture(IFlowCaptureReferenceOperation reference)
+            {
+                if (reference.Language != LanguageNames.CSharp)
+                {
+                    return false;
+                }
+
+                IOperation containingArgument = reference;
+                do
+                {
+                    containingArgument = containingArgument.Parent;
+                } while (containingArgument is not (null or IArgumentOperation));
```

Comment:
>} while (containingArgument is not (null or IArgumentOperation)); [](http://example.com/codeflow?start=16&length=65)

This looks like a violation of a style. Shouldn't braces be on their own lines?

Reply:
I use `do/while` loops so infrequently I haven't broken my Java habits with them.